lib/util/md5-hash.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+var crypto = require( "crypto" );
```

Comment:
Dont we need this package inside our dependency? if yes, the how is the unit test passing?

Also can you remove spaces around it just for consistency sake.